packages/http/src/utils/parseResponse.ts

```diff
@@ -19,7 +19,7 @@ export const parseResponseBody = (
   );
 
 export const parseResponseHeaders = (headers: Dictionary<string[]>): Dictionary<string> =>
-  mapValues(headers, hValue => hValue.join(' '));
+  mapValues(headers, hValue => hValue.join(','));
```

Comment:
This is probably the good change — but I'd say we only need to apply this on the `Set-Cookie` header instead of doing it for all of them. `mapValues` is likely offering an overload to also get the key and react accordingly

Reply:
The reason I didn't initially only special case `set-cookie` was because of the wording in rfc2616, as far as I can tell joining with a comma is right thing to do by default - a space would be the exception to the general rule.

I could be wrong, do you have a link to more/newer info about the correct way to multiple header fields into one?

Reply:
I need to take a deeper look into; the reason is that it seems like such rule applies only for "state" headers that is Cookies, Caching and some other that now I do not remember.

You can leave it as it is. I'll check it out later or tomorrow and then get back to you.